frontend/app/views/spree/products/show.html.erb

```diff
@@ -16,9 +16,10 @@
               <span aria-hidden="true">&#x2715;</span>
             </button>
             <div class="d-flex justify-content-center align-items-center product-details-images h-100">
-              <% if @product.variant_images.size > 1 %>
+              <% if @product_images.length > 1 %>
```

Comment:
wdyt about change `length` to `size` ?

Reply:
@przemosk this images collection is already loaded into memory, so it doesn't matter

Reply:
besides that, they are aliases 😉